tests/test_misc.py

```diff
@@ -2606,3 +2607,18 @@ def test_datastore_keylist(node_factory):
                                                                          'string': 'ab2val2',
                                                                          'generation': 1,
                                                                          'hex': b'ab2val2'.hex()}]}
+
+
+def test_checkmessage_pubkey_not_found(node_factory):
+    l1 = node_factory.get_node()
+
+    msg = "testcase to check new rpc error"
+    pubkey = "03be3b0e9992153b1d5a6e1623670b6c3663f72ce6cf2e0dd39c0a373a7de5a3b7"
+    zbase = "d66bqz3qsku5fxtqsi37j11pci47ydxa95iusphutggz9ezaxt56neh77kxe5hyr41kwgkncgiu94p9ecxiexgpgsz8daoq4tw8kj8yx"
+
+    with pytest.raises(RpcError):
```

Comment:
Make sure you check error matches what you expect, using `match="not found in the graph, expected pubkey is {}".format(pubkey)`!  Otherwise anything could trigger the error....